must-gather/collection-scripts/gather_clusterscoped_resources

```diff
@@ -64,3 +64,20 @@ for command_desc in "${commands_desc[@]}"; do
      COMMAND_OUTPUT_FILE=${BASE_COLLECTION_PATH}/cluster-scoped-resources/oc_output/desc_${command_desc// /_}
     { oc describe "${command_desc}"; } >> "${COMMAND_OUTPUT_FILE}"
 done
+
+kenel_collection() {
+        oc debug nodes/"${node}" --to-namespace="${ns}" -- bash -c "sleep 5m" &
+        while [ "$(oc get pods -n openshift-storage | grep "${node//./}-debug" | awk '{print $2}')" != "1/1" ] ; do
+            sleep 3
+            echo "waiting for the debug pod to be in ready state"
+        done
+        oc rsync -n "${ns}" "$(oc get pods -n "${ns}"| grep "${node//./}-debug"| awk '{print $1}')":/var/log/ "${KERNEL_OUTPUT_DIR}"
+    }
+
+# Collect kernel level logs
+for node in $(oc get nodes -l cluster.ocs.openshift.io/openshift-storage='' --no-headers | awk '/Ready/{print $1}'); do
+    printf "collecting kernel logs  from node %s \n"  "${node}"
+    KERNEL_OUTPUT_DIR=${BASE_COLLECTION_PATH}/kernel_${node}
```

Comment:
It would make it easier to see all the logs on a node in one place if we save the node logs in its own separate directory. For instance:

${BASE_COLLECTION_PATH}/${node}/kernel

If there are other node specific logs, we could create a similar directory structure for those as well.
Would this make sense in a k8s cluster scenario?

Reply:
That makes sense, we can consider that. Once I'll update the PR, I'll make those changes aswell.

Reply:
That actually depends on how people actually use the logs. You might want to check with support first - if they find it easier to look at logs on a per node basis, such a change would make sense.

Reply:
Yup. I will check with them. But first, we have to fix the logic that we are having issues with.

Reply:
For node data we will keep the data restricted to clustered scope dir.